Feat: Add "show" to pref-adjust (#1375)
DFHack/scripts: docs/pref-adjust.rst

===========

.. dfhack-tool::

>     :summary: Get the preferences of a dwarf, Set the preferences of a dwarf to a designated profile.

Reviewer: ```suggestion
    :summary: See the preferences of a dwarf or set them to a designated profile.
```

>     :tags: fort armok units preferences

Reviewer: ```suggestion
    :tags: fort armok units
```

Reviewer: You can only choose from the existing set of tags (documented here: https://docs.dfhack.org/en/latest/docs/Tags.html )

Usage
-----

``pref-adjust list``
    List all types of preferences. No changes will be made to any units.
``pref-adjust show``

>     Show the preferences of a unit.

Reviewer: ```suggestion
    Show the preferences of the selected unit.
```

``pref-adjust all|goth_all|clear_all``

>     Changes/clears preferences for all units.

Reviewer: ```suggestion
    Changes/clears preferences for all citizens.
```

``pref-adjust one|goth|clear``
    Changes/clears preferences for the currently selected dwarf.

Examples
--------

``pref-adjust all``

>     Change preferences for all units to an ideal.

Reviewer: ```suggestion
    Change preferences for all citizens to an ideal.
```

Reviewer: when we use "units" in these docs, we mean all units on the map, not just citizens.
